pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -516,6 +527,11 @@ func findPriorBackupNames(ctx context.Context, store cloud.ExternalStorage) ([]s
 	if err != nil {
 		return nil, errors.Wrap(err, "reading previous backup layers")
 	}
+	prevNew, err := store.ListFiles(ctx, "[0-9]*/[0-9]*.[0-9][0-9]/"+BackupNewManifestName)
```

Comment:
Since this wasn't here in 20.1 - I think this prevents us from upgrading the name w/o writing both names of the manifest in 20.2 since a 20.1 cluster may not see a previous incremental created by a 20.2 node.

Reply:
when does a 20.1 node need to see a 20.2 incremental?

Reply:
I think it would need to in the following case:
Consider a mixed-version cluster with 20.1 and 20.2 nodes. A 20.2 node could write an incremental backup to an append-able backup directory. Then, a later backup issued on a 20.1 node could go looking for the 20.2 incremental.

Alternatively, if a 20.2 cluster is not finalized, then the nodes could still be rolled back to 20.1 could try and back up on top of a 20.2 incremental.

Reply:
I think it'd be _okay_ since the 20.1 node would just ignore the 20.2 one -- it might get a TTL error, but that's probably OK. we don't say that new backups need to be readable by older nodes.

Reply:
Sounds good - I'll put up a renaming PR then. It might be worth calling out that limitation somewhere in the docs because it might be surprising to the user.